azext_edge/edge/providers/stats.py

```diff
@@ -88,7 +95,9 @@ def get_stats(
             stats = dict(sorted(_clean_stats(raw_metrics).items()))
             if not watch:
                 return stats
-            logger.warning(f"Refreshing every {refresh_in_seconds} seconds. Use ctrl-c to terminate stats watch.\n")
```

Comment:
No reason for this to change this, it takes up more lines when its 117 col length.